resources/assets/lib/chat/input-box.tsx

```diff
@@ -40,10 +37,17 @@ export default class InputBox extends React.Component<Props> implements Dispatch
   constructor(props: Props) {
     super(props);
 
-    dispatcher.register(this);
-
     makeObservable(this);
 
+    disposeOnUnmount(
+      this,
+      autorun(() => {
+        if (core.windowFocusObserver.hasFocus) {
+          this.focusInput();
```

Comment:
it has always been the case but I just noticed opening quick search when on chat page and then switching focus out and back in results in being able to type in chat while quick search is still open :upside_down_face: